src/vuegraf/vuegraf.py

```diff
@@ -138,6 +138,29 @@ def extractDataPoints(device, usageDataPoints):
                     usageDataPoints.append(createDataPoint(account, chanName, watts, timestamp, True))
                     index += 1
 
+def extractHistoricalDataPoints(device, startTime, stopTime, usageDataPoints):
+    excludedDetailChannelNumbers = ['Balance', 'TotalUsage']
+    minutesInAnHour = 60
+    wattsInAKw = 1000
+
+    for chanNum, chan in device.channels.items():
+        if chanNum in excludedDetailChannelNumbers:
+            continue
+
+        if chan.nested_devices:
+            for gid, nestedDevice in chan.nested_devices.items():
+                extractHistoricalDataPoints(nestedDevice, startTime, stopTime, usageDataPoints)
+
+        chanName = lookupChannelName(account, chan)
+        usage, usage_start_time = account['vue'].get_chart_usage(chan, startTime, stopTime, scale=Scale.MINUTE.value, unit=Unit.KWH.value)
+        index = 0
+        for kwhUsage in usage:
+            if kwhUsage is not None:
+                timestamp = startTime + datetime.timedelta(minutes=index)
+                watts = float(minutesInAnHour * wattsInAKw) * kwhUsage
+                usageDataPoints.append(createDataPoint(account, chanName, watts, timestamp, True))
```

Comment:
I noticed the `True` arg is being passed into createDataPoint() here. That indicates that this is _detailed_ data, which is a way for users to distinguish minute data vs second (detailed) data. Since this history data is per-minute, it might make more sense to pass `False` here instead of `True`.

Reply:
Thanks for catching that!